systemtests/CMakeLists.txt

```diff
@@ -615,6 +622,22 @@ else()
   list(APPEND SYSTEM_TESTS_DISABLED xattr)
 endif()
 
+if(AUTOCHANGER_DEVICES_FOUND)
+  find_program(MT mt)
+  find_program(MTX mtx)
```

Comment:
MTX is already found in cmake/BareosFindPrograms.cmake. It would make sense to also find MT there .